src/main/java/com/aws/greengrass/lifecyclemanager/Kernel.java

```diff
@@ -679,4 +681,8 @@ public String deTilde(String filename) {
     public List<String> getSupportedCapabilities() {
         return SUPPORTED_CAPABILITIES;
     }
+
+    public String getNucleusVersion() {
```

Comment:
This seems sketchy. Use the better method: https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/7d44f908703b3dba9a1782aec8ed6cd15a3ea3b5/src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java#L771

Reply:
Thanks